int32 key
string name
float64[2] start
float64[2] end
